_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -281,29 +281,35 @@ def check_bad_val(self, df_to_test, nameformat, signal_type, report):
 
         if percent_option:
             if not df_to_test[(df_to_test['val'] > 100)].empty:
+                bad_values = df_to_test[(df_to_test['val'] > 100)]['val'].unique()
```

Comment:
nice touch with the `.unique()` filter!  should we do that for all the messages that print lists of values?